napari/components/viewer_model.py

```diff
@@ -407,6 +409,7 @@ def add_image(
         blending='translucent',
         visible=True,
         path=None,
+        axis_labels=None,
```

Comment:
as noted above I don't think `axis_labels` should pass through `add_image` at this point, we might want to revisit this stuff when we do the xarray stuff, but for this PR let's keep all the `axis_labels` stuff happening within the `Dims` on the viewer_model